fluent-bundle/src/resolve.rs

```diff
@@ -35,6 +36,9 @@ pub struct Scope<'bundle, R: Borrow<FluentResource>> {
     args: Option<&'bundle FluentArgs<'bundle>>,
     /// Local args
     local_args: Option<FluentArgs<'bundle>>,
+    /// The running count of resolved placeables. Used to detect the Billion
+    /// Laughs and Quadrtic Blowup attacks.
+    placeables: usize,
```

Comment:
`u8` should be enough, no?

Reply:
Yes. I briefly considered `u8` but went for `usize` since it was what `MAX_PLACEABLE_LENGTH` was using. I'll change both to `u8`.